src/query/storages/parquet/src/parquet_part.rs

```diff
@@ -23,6 +23,7 @@ use common_arrow::parquet::compression::Compression;
 use common_arrow::parquet::indexes::Interval;
 use common_catalog::plan::PartInfo;
 use common_catalog::plan::PartInfoPtr;
+use common_catalog::table::ColumnId;
```

Comment:
Not modifying the codes in crate `common-storages-paruqet` is ok. The schema of `read_parquet` will not be changed.